examples/framesjs-starter/README.md

```diff
@@ -17,7 +17,7 @@ If running from the frames.js repository itself:
 
 4. Edit `app/page.tsx`
 
-5. Visit [http://localhost:3000/debug](http://localhost:3000/debug) to debug your frame.
+5. Visit [http://localhost:3010/](http://localhost:3010/) to debug your frame.
```

Comment:
cc @stephancill

Reply:
when we run the debugger, does it open this url automatically? if not, perhaps it should

Reply:
So first we need to release this to NPM and then we can do exactly what you want.